### DISCUSSION TITLE
how to resolve redis dns correctly using ScaledObject namespace rather than keda-operator namespace?

### DISCUSSION THREAD

**Discussion by yifan** [kedacore/keda#977] [Q&A / Need Help]
In ScaledObject we specify dns for hosts of many services, KAFKA, REDIS or RabbitMQ. It is right to assume those dns are being resolved in the same namespace as ScaledObject, not from the namespace of keda-operator.

### Use-Case

I have a set of deployments and ScaledObjects for these deployments. Also I have a redis server. I can deploy all these resources in namespace. I would then deploy exactly the same deployments, redis and ScaledObjects in a different namespace if I need to.

**Comment by Loki-Afro** [comment]
same inconvenience here too
